test_conformance/SVM/test_pointer_passing.cpp

```diff
@@ -50,7 +50,7 @@ int test_svm_pointer_passing(cl_device_id deviceID, cl_context context2, cl_comm
   test_error(error,"clCreateKernel failed");
 
   size_t bufSize = 256;
-  char *pbuf = (char*) clSVMAlloc(context, CL_MEM_READ_WRITE, sizeof(cl_uchar)*bufSize, 0);
+  cl_uchar *pbuf = (cl_uchar*) clSVMAlloc(context, CL_MEM_READ_WRITE, sizeof(cl_uchar)*bufSize, 0);
```

Comment:
Could we just rename this entirely? Its very confusing and easy to make a mistake having both `pbuf` and `pBuf`.